src/__stories__/boxed-story.tsx

```diff
@@ -1,42 +1,32 @@
 import * as React from 'react';
-import {Box, Boxed, Stack, Text8, skinVars} from '..';
-import {ThemeVariant} from '../theme-variant-context';
-import {useCheckbox} from './helpers';
+import {Box, Boxed, Text8, ResponsiveLayout} from '..';
 
 export default {
     title: 'Components/Primitives/Boxed',
+    parameters: {fullScreen: true},
 };
 
-export const Default: StoryComponent = () => {
-    const [isInverseOutside, inverseOutsideCheckbox] = useCheckbox('Inverse outside', false);
-    const [isInverseInside, inverseInsideCheckbox] = useCheckbox('Inverse inside', false);
+type Args = {
+    inverseInside: boolean;
+    inverseOutside: boolean;
+};
 
+export const Default: StoryComponent<Args> = ({inverseInside, inverseOutside}) => {
     return (
-        <>
-            <Stack space={16}>
-                {inverseOutsideCheckbox}
-                {inverseInsideCheckbox}
-            </Stack>
-            <ThemeVariant isInverse={isInverseOutside}>
-                <div
-                    data-testid="boxed"
-                    style={{
-                        backgroundColor: isInverseOutside
-                            ? skinVars.colors.backgroundBrand
-                            : skinVars.colors.background,
-                    }}
-                >
+        <ResponsiveLayout dataAttributes={{testid: 'boxed'}} fullWidth isInverse={inverseOutside}>
+            <Box padding={16}>
```

Comment:
this will add a double horizontal padding, because ResponsiveLayout already adds the needed horizontal padding. Replace with:
```suggestion
            <Box paddingY={16}>
```